sandbox.sh

```diff
@@ -0,0 +1,205 @@
+    current_block_hash = 0x;
+    current_block_height = 0;
+    current_state_hash = 0x;
+    current_handles_hash = 0x;
+    current_validators = [
+EOF
+    ## this iteration is done here just to ensure the indentation
+    for VALIDATOR in ${VALIDATORS[@]}; do
+      ADDRESS=$(echo $VALIDATOR | awk -F';' '{ print $4 }')
+      echo "      (\"$ADDRESS\": key_hash);"
+    done
+    cat <<EOF
+    ];
+  };
+  vault = {
+    known_handles_hash = (Big_map.empty : vault_known_handles_hash_set);
+    used_handles = (Big_map.empty : vault_used_handle_set);
+    vault = (Big_map.empty : vault);
+  }
+}
+EOF
+  )
+
+  consensus="./tezos_interop/consensus.mligo"
+  storage=$(ligo compile storage "$consensus" "$storage")
+  contract=$(ligo compile contract $consensus)
+
+  message "Originating contract"
+  sleep 2
+  tezos-client --endpoint $RPC_NODE originate contract "consensus" \
+    transferring 0 from myWallet \
+    running "$contract" \
+    --init "$storage" \
+    --burn-cap 2 \
+    --force
+
+  for VALIDATOR in ${VALIDATORS[@]}; do
+    i=$(echo $VALIDATOR | awk -F';' '{ print $1 }')
+    FOLDER="$DATA_DIRECTORY/$i"
+    validators_json >"$FOLDER/validators.json"
+    trusted_validator_membership_change_json >"$FOLDER/trusted-validator-membership-change.json"
+  done
+
+  message "Getting contract address"
+  TEZOS_CONSENSUS_ADDRESS="$(tezos-client --endpoint $RPC_NODE show known contract consensus | grep KT1 | tr -d '\r')"
+
+  message "Configuring Deku nodes"
+  for VALIDATOR in ${VALIDATORS[@]}; do
+    i=$(echo $VALIDATOR | awk -F';' '{ print $1 }')
+    FOLDER="$DATA_DIRECTORY/$i"
+
+    sidecli setup-tezos "$FOLDER" \
+      --tezos_consensus_contract="$TEZOS_CONSENSUS_ADDRESS" \
+      --tezos_rpc_node=$RPC_NODE \
+      --tezos_secret="$SECRET_KEY" \
+      --unsafe_tezos_required_confirmations 1
+  done
+}
+
+stop() {
+  if [[ $(docker ps | grep my-sandbox) ]]; then
+    docker kill my-sandbox
+    rm -rf "$DATA_DIRECTORY/**"
+    echo "Stopped the sandbox and wiped all state."
+  fi
+}
+
+start_node() {
+  stop
+  message "Starting sandbox"
+  docker run --rm --name my-sandbox --detach -p 20000:20000 \
+    tqtezos/flextesa:20210602 granabox start
+  sleep 3
+  message "Sandbox started"
+  message "Configuring Tezos client"
+  tezos-client --endpoint $RPC_NODE bootstrapped
+  tezos-client --endpoint $RPC_NODE config update
+  tezos-client --endpoint $RPC_NODE import secret key myWallet "unencrypted:$SECRET_KEY" --force
+}
+
+help() {
+  # FIXME: fix these docs
+  echo "$0 automates deployment of a Tezos testnet node and setup of a Deku cluster."
+  echo ""
+  echo "Usage: $0 start|stop"
+  echo "Commands:"
+  echo "start"
+  echo "  Does the following:"
+  echo "  - Starts a Tezos sandbox network with Flextesa."
+  echo "  - Generates new validator identities."
+  echo "  - Deploys a new contract to the Tezos sandbox configured to use these validators."
+  echo "stop"
+  echo "  Stops the Tezos node"
+}
+
+case "$1" in
+start)
```

Comment:
I would say rename the commands to `setup` and `prune`. As they're not safe commands, while `start` and `stop` should never destroys data.